### PR TITLE
Duplicate Detection while extending the Pipeline #364

### PR DESCRIPTION
**Pull request by rajkiranrbala**
Includes resource_instance_id and consumer_id as a part of org aggregated document. Addresses issue https://github.com/cloudfoundry-incubator/cf-abacus/issues/364

### CODE REVIEW

File: test/aggregation/reporting/src/test/test.js [hunk 1]
```diff
@@ -562,6 +563,8 @@ describe('abacus-usage-reporting-itest', () => {
       id: dbclient.kturi(oid(o), end + u),
       organization_id: oid(o),
       account_id: '1234',
+      resource_instance_id: 'rid',
+      cid: 'cid',
```
Comment (jsdelfino) [inline test/aggregation/reporting/src/test/test.js]: Shouldn't this be ```consumer_id: 'cid'```?

File: lib/aggregation/aggregator/src/index.js [hunk 2]
```diff
@@ -410,12 +410,17 @@ const aggregate = function *(aggrs, u) {
...
+      end: u.end,
+      resource_instance_id: u.resource_instance_id,
+      consumer_id: u.consumer_id
     }) :
     extend(newOrg(u.organization_id), {
+      account_id: u.account_id,
       start: u.start,
       end: u.end,
-      account_id: u.account_id
+      resource_instance_id: u.resource_instance_id,
```
Comment (jsdelfino) [inline lib/aggregation/aggregator/src/index.js]: Shouldn't the ```start```, ```end``` and ```resource_instance_id``` fields be added to the consumer level aggregated usage doc as well? If the intent is to help a downstream app do dup detection, then AIUI an app interested in consumer level aggregation would need these fields too in that doc, right?

**Comment by jsdelfino** [comment]
The latest changes in this pull request break the build (see above) and I'm seeing a build break when building locally as well. Any idea?

**Comment by hsiliev** [comment]
As far as I can see the Travis build was failing due to `cf-bridge` and `cf-renewer` tests.

I created https://github.com/cloudfoundry-incubator/cf-abacus/issues/366 to investigate and fix this.

**Comment by rajkiranrbala** [comment]
@jsdelfino Issue #336 was fixed by @hsiliev and the Travis build succeeds after that.

**Pull request merged**